rocky/reports/templates/partials/export_report_settings.html

```diff
@@ -13,115 +12,65 @@ <h2>{% translate "Report name" %}</h2>
                                     <td>
-                                        {% component "modal" size="dialog-medium" %}
-                                        {% fill "trigger" %}
                                         <button type="button"
-                                                id="edit-name-icon"
-                                                class="icon ti-edit action-button modal-trigger">
-                                            Edit {{ report_name }}
-                                        </button>
-                                    {% endfill %}
-                                    {% fill "header" %}
-                                    <h2>{% translate "Edit " %}{{ report_name }}</h2>
-                                {% endfill %}
-                                {% fill "content" %}
-                                <p>
-                                    {% blocktranslate %}
-                                                            Give your report a custom name and optionally add the reports' reference date
-                                                            to the name. To do so you can select an option below.
-                                                        {% endblocktranslate %}
-                                </p>
-                                <fieldset>
-                                    <legend>{% translate "Report name settings." %}</legend>
-                                    <input type="hidden"
-                                           class="old-report-name"
-                                           name="old-report-name"
-                                           value="{{ report_id }}">
-                                    <label for="report-name">{% translate "Report name" %}</label>
-                                    <br />
-                                    <span class="nota-bene">{% translate "This field is required." %}</span>
-                                    <div>
-                                        <input id="report-name"
-                                               class="new-report-name"
-                                               name="new-report-name"
-                                               type="text"
-                                               value="{{ report_name }}"
-                                               required>
-                                    </div>
-                                    <br />
-                                    <div>
-                                        <label for="reference-date">{% translate "Add reference date" %}</label>
-                                        <select class="reference-date" name="reference-date">
-                                            <option disabled selected value="">-- {% translate "Select option" %} --</option>
+                                                class="icon ti-arrow-back-up action-button reset-button"></button>
```

Comment:
This button needs an accessible name, preferably using an `aria-label` to shadow the icon's pseudo content:

```suggestion
                                                class="icon ti-arrow-back-up action-button reset-button" aria-label="Reset"></button>
```

Alternatively, you could move the icon to a nested aria-hidden span and add a visually hidden accessible text alongside it, i.e.:

```html
<button class="reset-button">
  <span class="icon ti-arrow-back-up" aria-hidden="true">
  </span><span class="visually-hidden">Reset</span>
</button>
```